docs/graphql/manual/actions/index.rst

```diff
@@ -20,8 +20,9 @@ enrichment from external sources and any other complex business logic.
 
 .. admonition:: Supported from
 
-   Actions are currently available for preview in the pre-release versions of
-   ``v1.2.0``
+   - Actions are currently available for preview in the pre-release versions of ``v1.2.0``
+
+   - Actions are only supported for ``Postgres versions 10 or higher``.
```

Comment:
I wouldn't mind if this Postgres version thing was in a separate admonition. Because it is kind of a permanent limitation.

Reply:
I moved it up because it's hard to come across if we put it at the bottom. It's something people should know upfront when they are evaluating actions. And two admonitions one below the other looks sorta ugly.

Reply:
let's keep it standard. @marionschleifer what's the standard way of mentioning availability?

Reply:
@tirumaraiselvan I put it on the bottom to be consistent how we inform about Postgres version on the getting started guide: https://hasura.io/docs/1.0/graphql/manual/getting-started/index.html.

Reply:
I think here it is fine calling it out upfront. High chance of missing it otherwise. I am in favour of using 2 separate admonitions. The 2 limitations are kind of unrelated. One is a hasura version thing vs a permanent limitation. Adding a separate note about the PG limitation like in gettting started makes sense.